README.md

```diff
@@ -202,6 +202,25 @@ storj.makePublic(bucketId, ["PULL", "PUSH"], function(e) {
 })
 ```
 
+### `storj.updateBucketById(bucketId, updateInfo, function cb(e) {})`
```

Comment:
This is nit-picky, but can we change the method to `updateBucket`? We have a pattern going that is `[verb][noun]()` and this breaks that.

Later on we can overload this method to accept names, ids, etc. and derive the type from the form of the first argument.